source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +477,90 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of event names to entity names.
+    If provided the test runner MUST:
+    
+      - For each entity name, create the respective entity with a type of
+        "event list". If multiple event names map to the same entity name,
+        exactly one entity MUST be created. If the entity already exists
+        (such as from a previous ``storeEventsAsEntities`` declaration from
+        another client), the test runner MUST raise an error.
+      - Set up an event subscriber for each event named. The event subscriber
+        MUST serialize the events it receives into a document and append
+        the document to the array stored in the specified entity.
+        The fields in the document MUST be as follows:
+        
+        - For ``CommandStartedEvent``:
```

Comment:
I'd suggest dropping this bullet as well. Bullets for the fields below make sense, though.

Optionally, you can add an earlier sentence or list that enumerates the supported event types. For example:

The following event names are supported:

 * From the [Command Monitoring](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/command-monitoring.rst#api) spec: CommandStartedEvent, ...
 * From the [CMAP](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events) spec: PoolCreatedEvent, ...